packages/rpc-core/src/response-patcher-allowed-numeric-values.ts

```diff
@@ -59,6 +59,7 @@ export const ALLOWED_NUMERIC_KEYPATHS: Partial<
         ['value', 'data', 'parsed', 'info', 'votes', KEYPATH_WILDCARD, 'confirmationCount'],
     ],
     getBlockTime: [[]],
+    getInflationRate: [['foundation', 'total', 'validator']],
```

Comment:
```suggestion
    getInflationRate: [['foundation'], ['total'], ['validator']],
```

I think you want this since each field is top-level

Reply:
Good catch. Done